src/style.css

```diff
@@ -1,3 +1,146 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+@media screen and (min-width: 1024px) {
+  header {
+    display: flex;
+    position: relative;
+    z-index: 1;
+    align-items: center;
+    padding: 0 50px;
+    justify-content: space-between;
+    background-color: #fff;
+  }
+
+  .header__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  .nav__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+
+  .nav__item {
+    margin-left: 20px;
+  }
+
+  .nav__item:first-child {
+    margin-left: 0;
+  }
+
+  .nav__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    font-size: 12px;
+    font-weight: 500;
+    white-space: nowrap;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #000;
+    transition: color 0.3s;
+  }
+
+  .nav__link:hover {
+    color: #00acdc;
+  }
+
+  .nav__link.is-active {
+    color: #00acdc;
+  }
+
+  .nav__list li.nav__item a {
+    position: relative;
+  }
+
+  .nav__list li.nav__item a.is-active::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 4px;
+    background-color: #00acdc;
+    border: #00acdc;
+    border-radius: 10%;
+  }
+
+}
+
+@media screen and (min-width: 1200px) {
```

Comment:
please remove this `@media` and all CSS above as it doesn't change anything. you just duplicated the styles